app/code/Magento/Downloadable/Observer/SaveDownloadableOrderItemObserver.php

```diff
@@ -150,6 +155,8 @@ public function execute(\Magento\Framework\Event\Observer $observer)
                         )->setNumberOfDownloadsBought(
                             $numberOfDownloads
                         )->setStatus(
+                            1 == $statusToEnable ?
```

Comment:
Please, get value from `\Magento\Sales\Model\Order\Item::STATUS_PENDING` instead of hardcoded ('1') value and use the strict comparison operator `===`

Reply:
I implemented all of the requested changes except for the strong comparison. If you var_dump the constant and the value returned from the admin setting, you can see that the constant is an integer while the admin setting value is a string.